Gems/Atom/Feature/Common/Code/Source/RayTracing/RayTracingPass.cpp

```diff
@@ -6,26 +6,34 @@
  *
  */
 
-#include <AzCore/Asset/AssetCommon.h>
-#include <AzCore/Asset/AssetManagerBus.h>
 #include <Atom/RHI/CommandList.h>
+#include <Atom/RHI/DeviceDispatchRaysItem.h>
+#include <Atom/RHI/DevicePipelineState.h>
+#include <Atom/RHI/DispatchRaysItem.h>
 #include <Atom/RHI/Factory.h>
 #include <Atom/RHI/FrameScheduler.h>
-#include <Atom/RHI/DeviceDispatchRaysItem.h>
 #include <Atom/RHI/RHISystemInterface.h>
-#include <Atom/RHI/DevicePipelineState.h>
 #include <Atom/RHI/RHIUtils.h>
-#include <Atom/RPI.Reflect/Pass/PassTemplate.h>
-#include <Atom/RPI.Reflect/Shader/ShaderAsset.h>
 #include <Atom/RPI.Public/Base.h>
 #include <Atom/RPI.Public/Pass/PassUtils.h>
 #include <Atom/RPI.Public/RPIUtils.h>
 #include <Atom/RPI.Public/RenderPipeline.h>
 #include <Atom/RPI.Public/Scene.h>
 #include <Atom/RPI.Public/View.h>
+#include <Atom/RPI.Reflect/Pass/PassTemplate.h>
+#include <Atom/RPI.Reflect/Shader/ShaderAsset.h>
+#include <AzCore/Asset/AssetCommon.h>
+#include <AzCore/Asset/AssetManagerBus.h>
+#include <RayTracing/RayTracingFeatureProcessor.h>
 #include <RayTracing/RayTracingPass.h>
 #include <RayTracing/RayTracingPassData.h>
-#include <RayTracing/RayTracingFeatureProcessor.h>
+
+#ifndef INDIRECT_RENDERING_INCLUDE_GUARD_FOR_MISSING_INCLUDE_GUARD
+#define INDIRECT_RENDERING_INCLUDE_GUARD_FOR_MISSING_INCLUDE_GUARD
```

Comment:
Could just name it `INDIRECT_RENDERING_INCLUDE_GUARD`

Reply:
done